.gitignore

```diff
@@ -9,6 +9,7 @@ build/
 npm-debug.log
 node_modules/
 src/js/config/Config.dev.js
+src/js/config/Config.dev.ts
```

Comment:
Why do we need this? Can't we just remove the file when we switch branches?

Reply:
Could be annoying when switching between 1.11 and master in the same dir. I'd say we merge it and bump the release until the deadline. As it is not a blocker is it?

Reply:
Correct, not a blocker